src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,25 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            switch ($this->options['diff']) {
+                case false:
```

Comment:
if it's not possible to get it from `InputOption` then current way is fine for me. we have default case with exception